src/wizards/newspack/views/settings/advanced-settings/pwa-display-mode.tsx

```diff
@@ -0,0 +1,52 @@
+import { __ } from '@wordpress/i18n';
+import { SelectControl } from '@wordpress/components';
+
+import { Grid, Notice } from '../../../../../components/src';
+
+interface PwaDisplayModeProps extends ThemeModComponentProps< AdvancedSettings > {}
+
+export default function PwaDisplayMode( { data, isFetching, update }: PwaDisplayModeProps ) {
+	const displayModeOptions = [
```

Comment:
non-blocking nitpick: Since this is static, this can be defined outside of the component so it isn't redefined everytime this component is re-rendered.